data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -1531,6 +1524,41 @@ mod tests {
         assert_eq!(result, AgentResponse::from(0.8));
     }
 
+    #[test]
+    #[cfg_attr(miri, ignore)]
```

Comment:
Are you skipping miri here because you were seeing an access violation error with the tinybytes payload? If so, you can create the bytes from static like [here](https://github.com/DataDog/libdatadog/blob/962589d7316094430b90327c90dd60e4095642ae/trace-utils/src/msgpack_decoder/v04/decoder/mod.rs#L327).

Reply:
I'm skipping those tests because it appears that `libc::socket` call is not emulated in Miri, at least in the nightly version we're currently using.